iothub/device/src/ClientOptions.cs

```diff
@@ -19,9 +19,9 @@ public class ClientOptions
 
         /// <summary>
         /// The transport settings to use for all file upload operations, regardless of what protocol the device
-        /// client is configured with. All file upload operations take place over https. 
-        /// If FileUploadTransportSettings is not provided, then file upload operations will use the client certificates configured
-        /// in the transport settings set for the non-file upload operations.
+        /// client is configured with. All file upload operations take place over https.
+        /// If FileUploadTransportSettings is not provided, then file upload operations will use the same client certificates
+        /// configured in the transport settings set for client connect.
         /// </summary>
```

Comment:
should we add a comment to call out how users should set proxy?

Reply:
I mean, all we call out is client cert here, so I don't think we need to specify all the other things you'd need to set. Do you?